CryptoTechReminderSystem/Gateway/HarvestGateway.cs

```diff
@@ -42,11 +43,15 @@ public IList<HarvestDeveloper> RetrieveDevelopers()
             ).ToList(); 
         }
 
-        public IList<TimeSheet> RetrieveTimeSheets()
+        public IList<TimeSheet> RetrieveTimeSheets(DateTimeOffset dateFrom, DateTimeOffset dateTo)
         {
             _client.DefaultRequestHeaders.Authorization = new AuthenticationHeaderValue("Bearer", _token);
- 
-            var apiResponse = GetApiResponse("/api/v2/time_entries").Result;
+            
+            var dateFromFmt = dateFrom.ToString("yyyy-MM-dd");
```

Comment:
```suggestion
            var dateFromFormat = dateFrom.ToString("yyyy-MM-dd");
```
Think it's more matching of our naming convention throughout the project

Reply:
+1

Reply:
We have decided to inline the variables instead of renaming them.

Resolved in https://github.com/chubberlisk/crypto_tech/pull/23/commits/6355fa3c28df6cc7daa5e9ad887a6d7b73f3a6f9.